blueprints/codes/cur/cur_228/formula_2_21.py

```diff
@@ -0,0 +1,93 @@
+"""Contains formula 2.21 from CUR 228."""
+
+from blueprints.codes.cur.cur_228 import CUR_228
+from blueprints.codes.formula import Formula
+from blueprints.codes.latex_formula import LatexFormula
+from blueprints.type_alias import KN_M3, KPA, M
+from blueprints.validations import raise_if_negative
+
+
+class Form2Dot21MenardStiffness(Formula):
+    """Representation of equation 2.21 CUR 228."""
+
+    source_document = CUR_228
+    label = "2.21"
+    r_0: M = 0.3
+
+    def __init__(self, r: M, e_p: KPA, alpha: float) -> None:
```

Comment:
Use DIMENSIONLESS instead of float
```suggestion
from blueprints.type_alias import KN_M3, KPA, M, DIMENSIONLESS
from blueprints.validations import raise_if_negative


class Form2Dot21MenardStiffness(Formula):
    """Representation of equation 2.21 CUR 228."""

    source_document = CUR_228
    label = "2.21"
    r_0: M = 0.3

    def __init__(self, r: M, e_p: KPA, alpha: DIMENSIONLESS) -> None:
```